src/app/Guide/page.tsx

```
@@ -32,20 +32,25 @@ const Guide = () => {
-                  text={item.title}
-                  kind="bold24"
-                  numberOfLines={2}
-                  ellipsizeMode="tail"
-                />
-              </InfoWrapper>
-            </GuideItem>
-          ))}
+          {data &&
+            data.map((item: GuideItemType) => (
+              <GuideItem onPress={() => navigation.navigate('GuideDetail')}>
+                <GuideImage></GuideImage>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)